app/posts/teacher-success/2025-07-01-need-for-teacher-success.md

```diff
@@ -0,0 +1,44 @@
+---
+title: The need for a Teacher Success Service
+description: How investigating how to improve withdrawal rates from teacher training led to a bigger opportunity to support aspiring teachers
+date: 2025-07-01
+tags:
+  - withdrawals
+  - trainees
+  - training outcomes
+---
+
+## The starting problem
+
+The 2024 Get Ready to Teach squad was tasked with understanding why 10% of teacher trainees withdraw from their studies without achieving Qualified teacher status (QTS).
+
+Our intuition was that 10% withdrawal is probably reasonable for any higher education programme of study, but there is a clear opportunity to retain promising but struggling trainees and contribute efficiently to the government’s target of 6,500 additional teachers.  
+
+## The evolving problem
+
+Through rounds of qualitative and quantitative user research with providers and trainees that both achieved or did not achieve QTS, we uncovered additional trends.
+
+The most telling of these was that just over 50% of trainees who achieved QTS reported the experience having a detrimental effect on their mental wellbeing. This helped explain why as many as 20% of trainees who achieve QTS do not go on to teach in a state school (though more research is needed to better understand that statistic).  
+
+This led to the pitch document being created proposing a Trainee Teacher Support Hub. We thought this didn’t go far enough and believed we were missing an opportunity to go beyond providing reactive crisis intervention and proactively set trainee teachers up for career long success.  
+
+## Defining the compound outcome
+
+It became clear that getting trainees across the line to QTS at all costs was not meaningful in delivering enough good quality teachers to the sector. We wanted to help deliver more qualified teachers into their first state teaching job with the resilience they needed to enjoy a fulfilling career.
+
+This meant working across traditional team boundaries to deliver a service that can enable teachers at all stages, from their ITT year, into the Early career framework (ECF) and beyond. We therefore began to envision a service that, in the first instance, would enable and support trainees. In subsequent iterations, it would be personalised to the individual’s needs throughout their career.  
```

Comment:
```suggestion
This meant working across traditional team boundaries to deliver a service that can enable teachers at all stages, from their ITT year, into the early career framework (ECF) and beyond. We therefore began to envision a service that, in the first instance, would enable and support trainees. In subsequent iterations, it would be personalised to the individual’s needs throughout their career.  
```